codes/java/Chapter5/5.3/MaxInducedSubgraph.java

```diff
@@ -0,0 +1,102 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+public class MaxInducedSubgraph {
+
+	public static void main(String[] args) {
```

Comment:
1. Writing the algorithm in a standalone function.
2. The data needed should be transferred by the signature, no STDIO in the function.
3. The err should be throw instead of print out.